coredb-operator/src/statefulset.rs

```diff
@@ -339,21 +363,21 @@ async fn delete_sts(client: Client, sts: &StatefulSet) {
     };
 
     let _o = sts_api
-        .delete(&sts_name, &delete_params)
+        .delete(sts_name, &delete_params)
         .await
         .map_err(Error::KubeError);
 }
 
-async fn update_pvc(client: Client, sts: &StatefulSet, cdb: &CoreDB) {
-    let pvc_api: Api<PersistentVolumeClaim> =
-        Api::namespaced(client, &sts.clone().metadata.namespace.unwrap());
-
-    let sts_name = sts.clone().metadata.name.unwrap();
-    let pvc_name = format!("data-{sts_name}-0");
+async fn update_pvc(pvc_api: &Api<PersistentVolumeClaim>, sts_name: &str, pvc_name: &str, value: Quantity) {
```

Comment:
Made this function a bit more generic, so we can use it for updating each pvc individually.